Makefile

```diff
@@ -306,6 +312,16 @@ octant-antrea-ubuntu:
 	docker build --pull -t antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION) -f build/images/Dockerfile.octant.ubuntu .
 	docker tag antrea/octant-antrea-ubuntu:$(DOCKER_IMG_VERSION) antrea/octant-antrea-ubuntu
 
+.PHONY: flow-aggregator-ubuntu
```

Comment:
should we add this target along with `build-ubuntu` to get executed when we do `make`? Or expect folks to build image by using the target directly? @antoninbas what do you think?